localisation/00_Kingdom_names.csv

```diff
@@ -83,8 +83,8 @@ k_orsis;Orsis;Orsis;Orsis;;Orsis;;;;;;;;;x
 k_orsis_adj;Orsisian;;orsiser;;;;;;;;;;;x
 k_plunder_isle;Plunder Isle;Plunder Isle;Plunder Isle;;Plunder Isle;;;;;;;;;x
 k_plunder_isle_adj;Plunder Islean;;Plunder Isler;;;;;;;;;;;x
```

Comment:
This is just a suggestion as I don't like Islean, but maybe have Plunderlander instead? It doesn't sound great, but Islean is lazy. I've never heard of a group calling themselves x Isleans, but maybe I'm wrong.

Reply:
Or Plunderean maybe?

Reply:
I considered Plunderean, yeah. It's better than Islean for sure. My main reason for suggesting Plunderlander over Plunderean was that Plunder Isle is obviously a reference to the pirates on the isle, so I felt that Plunderlander was more of a settler/piratical name then Plunderean, which sounds more like what a native would call themselves. Sorry, that's rather convoluted. Both work.

Reply:
Maybe just **Plunder**? So, there will be **2nd Plunder Army**, **Plunder Fleet**.

Reply:
went with Plunderean